.github/workflows/ragstack-test.yml

```diff
@@ -36,20 +36,20 @@ jobs:
       - uses: nicoloboschi/setup-astradb@v1
         id: astra
         with:
-          token: ${{ secrets.TESTS_ASTRA_DEV_DB_TOKEN }}
+          token: ${{ secrets.ASTRA_DB_APPLICATION_TOKEN_DEV }}
```

Comment:
isn't this token related to the db you created previously? if so, it shoulnd't be able to create new DBs